src/web/mjs/connectors/Tapas.mjs

```diff
@@ -103,6 +76,7 @@ export default class Tapas extends Connector {
     }
 
     async _getPagesNovel(request) {
+        // Not updated since hakuneko doesn't support novels anyways.
```

Comment:
Novels are supported, otherwise this code wouldnt be here. People can copy paste novel url.

_getMangaFromURI (clipboard support) needs to be fixed so we can test that, even for manga only.

Reply:
tested and it seems to work after implementing clipboard support